apps/swap-service/.env.example

```
@@ -7,6 +7,8 @@ SERVICE_API_KEY=
 VITE_BEBOP_API_KEY=
 VITE_CHAINFLIP_API_KEY=
 VITE_NEAR_INTENTS_API_KEY=
+VITE_RELAY_API_KEY=
+VITE_ACROSS_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ACROSS_API_KEY key should go before the VITE_BEBOP_API_KEY key

(UnorderedKey)